src/api/app/controllers/webui/main_controller.rb

```diff
@@ -91,7 +91,11 @@ def add_news
       redirect_to(:action => 'index') and return
     end
     # TODO - make use of permissions.status_message_create
-    StatusMessage.create!(message: params[:message], severity: params[:severity], user: User.current)
+    begin
```

Comment:
Which doesn't make it a standard:-) And if most of existing code is already written in a different way it might even make the things messier.

That said, I changed it now.